.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -57,7 +74,7 @@ body:
     attributes:
       label: Conda config
       description: |
-        Let's see what your Conda config looks like. Run the following command from your command line and paste the output
+        Let's examine your conda configurations. Run the following command and paste the output in the field below.
 
         ```bash
         conda config --show-sources
```

Comment:
Should this command actually be `conda config --show`?

Reply:
conda config --show-sources  provides what is in the various .condarc files - which is often very helpful in tracking down a problem.  
conda config --show is also helpful though, I could see an argument to include both - but I think usually just the contents of the .condarc files is often enough to help debug someone's issue.

Reply:
I see. Thanks @csoja